garrysmod/lua/includes/extensions/entity.lua

```diff
@@ -4,6 +4,22 @@ local meta = FindMetaTable( "Entity" )
 -- Return if there's nothing to add on to
 if ( !meta ) then return end
 
+function meta:SetSpawnFlags( Flags )

```

Comment:
Nitpick: the argument name is capitalized here, and not capitalized in your other changes. It should all be lowercase.